auv_control.orogen

```diff
@@ -96,6 +96,10 @@ task_context 'BasePIDController' do
     # Under this Value the axis are not controled. Output is 0.
     property "variance_threshold", "double"
 
+    # This property defines the timeout for the pose_samples input port in seconds.
+    # 0 means that the timeout would be ignored.
+    property "timeout_pose", "double", 1
```

Comment:
Ok.
Since we are here, how about the Base's [timeout_in](https://github.com/rock-control/control-orogen-auv_control/blob/master/auv_control.orogen#L38) and `addCommandInput`'s [timeout](https://github.com/rock-control/control-orogen-auv_control/blob/master/auv_control.orogen#L65)?